minigalaxy/launcher.py

```diff
@@ -70,6 +70,9 @@ def get_windows_exe_cmd(game, files):
     exe_cmd = [""]
     prefix = os.path.join(game.install_dir, "prefix")
     os.environ["WINEPREFIX"] = prefix
+    command = game.get_info("command").split(" ")
+    var = game.get_info("variable").split(" ")
+    i = 0
```

Comment:
This can be done easier. You can use split() without any arguments. It will not produce empty strings in the list and will handle things like double spaces and tabs. But be aware that empty list can be result of such an operation. also i=0 is not needed here. For example:
command_list = game.get_info("command").split()
var_list = game.get_info("variable").split()

Reply:
Oh god, that's more easiest you're right. Thank you Master, still more things to learn with Python :-)
Change done.